wicker/schema/dataloading.py

```diff
@@ -87,6 +87,12 @@ def process_object_field(self, field: schema.ObjectField) -> Optional[Any]:
             return data
         return field.codec.decode_object(data)
 
+    def process_sf_variant_field(self, field: schema.VariantField) -> Optional[Any]:
```

Comment:
field type should not be tied with storage, here sf. I would suggest make more abstraction.
we can add a backend field to the schema definition, just as whether heavy duty. default s3, could be sf. if sf, we should call the SF specific decoder even for the same data type, aka numpy.

Reply:
@kds1010 , could we close this PR ?